File: be/src/main/java/com/example/campung/lankmark/dto/LandmarkDetailResponse.java
package com.example.campung.lankmark.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class LandmarkDetailResponse {
    private Long id;
    private String name;
    private String description;
    private String thumbnailUrl;
    private String imageUrl;
    private String category;
    private Double latitude;
    private Double longitude;
    private String currentSummary;
    private String summaryUpdatedAt;
    private String createdAt;
}

File: android/campung/app/src/main/java/com/shinhan/campung/data/remote/response/LandmarkDetailResponse.kt
package com.shinhan.campung.data.remote.response

data class LandmarkDetailResponse(
    val success: Boolean,
    val data: LandmarkDetailData
)

data class LandmarkDetailData(
    val id: Long,
    val name: String,
    val description: String?,
    val thumbnailUrl: String?,
    val imageUrl: String?,
    val category: String,
    val latitude: Double,
    val longitude: Double,
    val currentSummary: String?,
    val summaryUpdatedAt: String?,
    val createdAt: String
)